FileHandler.gd

```diff
@@ -0,0 +1,16 @@
+static func call_xml_converter(args: PoolStringArray):
+	var output: Array = []
+	print(args)
+	var result: int = OS.execute("./xml_converter/build/xml_converter", args, true, output, true)
```

Comment:
This can probably be a constant variable in FileHandler instead of a magic string here.

Also: does this work with the exported files? I think it is fine if not right now but we will need to test it to be sure.